src/test/java/SixPMComTestSuite/SixPMFeaturesTest.java

```diff
@@ -0,0 +1,88 @@
+        Actions action = new Actions(driver);
+        Random random = new Random();
+
+        driver.manage().window().maximize();
+        driver.get(homePage);
+
+        WebElement accessoriesElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(accessoriesXPath)));
+        action.moveToElement(accessoriesElem).build().perform();
+
+        WebElement aviatorsButtonElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(aviatorsButtonXPath)));
+        aviatorsButtonElem.click();
+
+        List <WebElement> listOfAllItems = wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(By.xpath(listOfAllItemsXPath)));
+
+        int countOfItems = listOfAllItems.size();
+        listOfAllItems.get(random.nextInt(countOfItems)).click();
+
+        WebElement addToBagElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(addToBagElemXPath)));
+        addToBagElem.click();
+
+        WebElement itemNameElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(itemNameElemActual)));
+        String actualItemName = itemNameElem.getText();
+        WebElement itemInfoElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(itemInfoElemActual)));
+        String actualItemInfo = itemInfoElem.getText();
+        WebElement itemPriceElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(itemPriceElemActual)));
+        String actualItemPrice = itemPriceElem.getText();
+
+        WebElement viewBugButtonElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(viewBugButtonElemXPath)));
+        viewBugButtonElem.click();
+
+        itemNameElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(itemNameElemExpected)));
+        String expectedItemName = itemNameElem.getText();
+        itemInfoElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(itemInfoElemExpected)));
+        String expectedItemInfo = itemInfoElem.getText();
+        itemPriceElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(itemPriceElemExpected)));
+        String expectedItemPrice = itemPriceElem.getText();
+
+        Assert.assertEquals(actualItemName,expectedItemName,"The actual item name does not match with expected item name");
+        Assert.assertEquals(actualItemInfo,expectedItemInfo,"The actual item info does not match with expected item info");
+        Assert.assertEquals(actualItemPrice,expectedItemPrice,"The actual item price does not match with expected item price");
+
+        WebElement itemQuantityDropDown = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(itemQuantityDropDownXpath)));
+        itemQuantityDropDown.click();
+        WebElement removeButtonFromDropDown = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(removeButtonFromDropDownXpath)));
+        removeButtonFromDropDown.click();
```

Comment:
These are post test actions, so they shouldn't be in test method.

Reply:
changed